.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,21 @@ Describe what this pull request changes, and why. Include implications for peopl
 - [ ] Consider running your code modifications in the included example app within `frontend-platform`. This can be done by running `npm start` and opening http://localhost:8080.
 - [ ] Consider testing your code modifications in another local micro-frontend using local aliases configured via [the `module.config.js` file in `frontend-build`](https://github.com/openedx/frontend-build#local-module-configuration-for-webpack).
 - [ ] Verify your commit title/body conforms to the conventional commits format (e.g., `fix`, `feat`) and is appropriate for your code change. Consider whether your code is a breaking change, and modify your commit accordingly.
+  - [ ] Verify intended release version at https://github.com/semantic-release/semantic-release#how-does-it-work
+
+**For Major Release Only**
+- [ ] Update dependency/peer dependency repo to support the breaking change
+  - [ ] https://github.com/openedx/frontend-component-footer
+    - Example: https://github.com/openedx/frontend-component-footer/pull/241
+  - [ ] https://github.com/openedx/frontend-component-header
+    - Example: https://github.com/openedx/frontend-component-header/pull/275
+  - [ ] https://github.com/edx/frontend-component-footer-edx
+    - Example: https://github.com/edx/frontend-component-footer-edx/pull/208
+  - [ ] https://github.com/edx/frontend-component-header-edx
+    1. [ ] update `@edx/frontend-enterprise-catalog-search`, `@edx/frontend-enterprise-logistration`, `@edx/frontend-enterprise-utils` to use the same peer dependency
+        - Example: https://github.com/openedx/frontend-enterprise/pull/278
+    2. [ ] upgrade the release package to `frontend-component-header-edx`
+        - Example: ....
```

Comment:
Is there no PR to header-edx, yet?

Reply:
it require `frontend-enterprise` to update first before `header-edx` can update the `enterprise package and create the pr.

Reply:
Alright, gotcha.  See my review comment, in any case: it probably makes this moot.